src/hooks/useAuth.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from "react"
+import type { User } from "../types"
+import { createFetcher, getLocalStorage, setLocalStorage } from "../utils"
+
+const REFRESH_TIMER = 60 * 1000 * 30; // 30 minutes in ms

```

Comment:
The refresh response includes a ExpiresIn key, it would be cool to use that in case it changed to be shorter than 30 mins

Reply:
Will do